README.md

```diff
@@ -1 +1,62 @@
-Dockerfile build Actions Status: <img src='https://img.shields.io/github/workflow/status/i13302/learn/push_image/main'>
+# Learing Experiment As Running contaiNer
+![build](https://img.shields.io/github/workflow/status/i13302/learn/push_image/main)
+![last-commit](https://img.shields.io/github/last-commit/i13302/learn)
+
+## Motivation
+This project is developed by whom are working at teaching assistant that is teach AI (Artificial Intelligence) in experimence class.   
+AI relating libraries are large and many , so installing their're not finished  in class time, and  futher, their install process are complex.  
+Therefore, we have to work suppotting install proccess at "service overtime" without pay.   
+Thus we have developed easier install process.  
+This Container is installed many AI relating libraries, by using Docker is  easy introduce.  
+Many students aren't to need worrying about install process, what you need is all in this container.  
```

Comment:
I want to add like `We contained simple sample code. So, you can immediately validation.`